src/mobile/src/ui/components/AnimatedComponent.js

```diff
@@ -57,7 +57,7 @@ class AnimatedComponent extends Component {
         this.screen = last(props.navStack);
     }
 
-    componentDidMount() {
+    componentWillMount() {
```

Comment:
```suggestion
    UNSAFE_componentWillMount() {
```
The name `componentWillMount` will stop working in React 17: https://reactjs.org/docs/react-component.html#unsafe_componentwillmount